specs/composer/show_all_packages.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: Show a list of available packages
```

Comment:
Mind making the name here and below clearer that this is related to composer? So `Show a list of available composer packages` instead?